<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gameland</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>851</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: black</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>761</width>
      <height>551</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>761</width>
      <height>551</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>761</width>
      <height>551</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { /* Панель с вкладками */
                background-color: white;
            }
QTabBar::tab {
background-color: black;
	font: 20pt &quot;OCR A Extended&quot;;
color: white;
  padding: 10px 20px;
  border-radius: 5px;
  cursor: pointer;
  box-shadow: 2px 2px 5px rgba(0, 0, 0, 0.3);
 width: 200px;
  height: 60px;
cursor: &quot;Open Hand&quot;;
}
</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Games</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cascadia Mono SemiBold</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>Games</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>641</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="games_lay"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Records</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cascadia Mono SemiBold</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Records</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>631</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="records_lay"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>My Profile</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>301</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cascadia Mono SemiBold</family>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Your Profile</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email: </string>
      </property>
     </widget>
     <widget class="QLabel" name="for_name">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="for_password">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="for_email">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Comic Sans MS</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="leave_account_btn">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>420</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cascadia Code SemiBold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; background: red</string>
      </property>
      <property name="text">
       <string>Leave account</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
